src/main/java/uk/gov/companieshouse/model/Constants.java

```diff
@@ -18,6 +19,10 @@ private Constants() {}
    public static final String PAYMENT_AVAILABLE_PAYMENT_METHOD = "credit-card";
    public static final String PAYMENT_CLASS_OF_PAYMENT = "data-maintenance";
 
+   /* CHIPS */
+   public static final int CHIPS_RETRY_DELAY_MINUTES = 30;
```

Comment:
This should be read from an environment variable, not a constant. I would extend ChipsConfig.java to enable this.